### PR TITLE
Get admin for each session

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: TrelloMigrate/Transform.fs [hunk 1]
```diff
@@ -98,16 +101,27 @@ module private SessionAndSpeaker =
         else None
 
     //Note: Currently ignoring any cards that don't have the title (name) filled out correctly. 
-    let tryCreate (card : BasicCard) = 
+    let tryCreate (admins : Admin []) (card : BasicCard) = 
         match tryParseCardName card.Name with
         | Some parsedCard -> 
+            let adminId =
```
Comment (pallenSL) [inline TrelloMigrate/Transform.fs]: I think you should cut this out into something like an AdminId.create()
Then the branch would just create the Some using 3x
Field = Something.createFrom parsedCard

File: TrelloMigrate/SrmImport.fs [hunk 2]
```diff
@@ -27,16 +30,27 @@ let private importSpeakers (sessionsAndSpeakers : SessionAndSpeaker [] ) (adminP
...
     profiles |> Array.iter (fun profile -> profile.Handles |> Array.iter Handle.post)
+
+let private setAdminIdOnSessions (importedAdmins : Admin []) (ss : SessionSpeakerAndTrelloIds) = 
+    let foundAdminId = 
+        match ss.AdminTrelloId with 
+        | Some trelloId -> 
+            importedAdmins 
+            |> Array.tryPick (fun admin -> if trelloId = admin.TrelloMemberId then Some admin.ProfileWithHandles.Profile.Id else None)
+        | None -> None
+    match foundAdminId with
```
Comment (pallenSL) [inline TrelloMigrate/SrmImport.fs]: Surely it must be possible to simplify this by assigning:
Status = if foundAdminId then "assigned" else None

File: TrelloMigrate/Transform.fs (hunk 1 above)
Comment (jebbinSCL) [inline TrelloMigrate/Transform.fs]: Updated

File: TrelloMigrate/SrmImport.fs (hunk 2 above)
Comment (jebbinSCL) [inline TrelloMigrate/SrmImport.fs]: Status is needed in both cases, but I've refactored in a similar way.

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**